pupil-spa/src/app/spoken-practice-question/spoken-practice-question.component.ts

```diff
@@ -15,6 +15,11 @@ import { QuestionRendered } from '../services/audit/auditEntry';
 export class SpokenPracticeQuestionComponent extends PracticeQuestionComponent implements OnInit, AfterViewInit, OnDestroy {
   private subscription: Subscription;
 
+  /**
+   * TBD
+   */
+  public showPracticeBox: boolean;
```

Comment:
Is this var needed?

Reply:
@jon-shipley Not needed any longer. Removed.